internal/app/cf-terraforming/cmd/generate.go

```diff
@@ -879,6 +880,106 @@ func generateResources() func(cmd *cobra.Command, args []string) {
 				if err != nil {
 					log.Fatal(err)
 				}
+			case "cloudflare_teams_list":
+				jsonPayload, _, err := api.ListTeamsLists(context.Background(), &cloudflare.ResourceContainer{Identifier: accountID}, cloudflare.ListTeamListsParams{})
+				if err != nil {
+					log.Fatal(err)
+				}
+				// creating our own listItems struct because Items nees to be list of string to match 
```

Comment:
check out the prior art on remapping responses - https://github.com/cloudflare/cf-terraforming/pull/669/files#diff-c8a271423d325a18f64e8eb426bae4c393315dfd1fbc9bad21916c9eead1647aL852